.github/workflows/verify.yaml

```
@@ -1,10 +1,15 @@
 name: Verify
 
-run-name: "Verify PR #${{ github.event.pull_request.number }}"
+description: |
```

actionlint (1.7.12)

[error] unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)